src/components/Calendar/calendar.css

```diff
@@ -6,6 +6,14 @@
     justify-content: center;
 }
 
+.rbc-header.dark-mode {
+    border-bottom: none;
+    color: white;
+    display: inline-flex;
+    align-items: center;
```

Comment:
Why does it need these `display`, `border-bottom`, `align-items`, and `justify-content` properties if it's just a modifier for dark mode? My intuition says that all you'd need to put here is the `color` property, so can you simplify this by removing the unnecessary properties or explain why they actually are necessary?